Automattic/wp-calypso
# Add onboarding squarespace importer

#### Changes proposed in this Pull Request

* Add a Squarespace importer
* Add support for favicon-s on the author mapping screen (added to `ImporterAuthorMapping`)
* Add a `CompleteScreen` and `ProgressScreen` components shared between routes
* Minor refactoring of importer components to pass `UrlData`

#### Testing instructions

* Open http://calypso.localhost:3000/start/importer/capture?siteSlug=SITE_SLUG
* Input a Squarespace URL such as https://squarespace.com/
* Select a Squarespace `.xml` export file; you can find one on _Drive > Caribou > Sample Export Files > Squarespace_
* Import the XML
* Check if the data have been imported

#### Important
The author list component is missing the favicon because Squarespace `analyze-call` returns `null`.
